lesson_27/pablo_lesson27/src/pages/Home/components/ProgramList/NewPrograms.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useState} from 'react';
+import {Link} from 'react-router-dom';
+
+const NewPrograms = () => {
```

Comment:
Really would not expect this component to go here. Would be better in its own subfolder of `pages`.